app/services/proofing/mock/ddp_mock_client.rb

```diff
@@ -20,9 +20,27 @@ class DdpMockClient < Proofing::Base
 
       TRANSACTION_ID = 'ddp-mock-transaction-id-123'
 
+      # Trigger the "REJECT" status
+      REJECT_STATUS_SSN = '666-77-8888'
```

Comment:
Can we add unit tests for these?

Reply:
Here's an example for a different mock proofer: https://github.com/18F/identity-idp/blob/d87550c9897ea082cd93e16907daf318880c94f1/spec/services/proofing/mock/resolution_mock_client_spec.rb